src/groups/bmq/bmqimp/bmqimp_sessionid.h

```diff
@@ -0,0 +1,82 @@
+// Copyright 2025 Bloomberg Finance L.P.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// bmqimp_sessionid.h                                             -*-C++-*-
+#ifndef INCLUDED_BMQIMP_SESSIONID
+#define INCLUDED_BMQIMP_SESSIONID
+
+//@PURPOSE: Provide the type for logging Application / Session instance number
+//
+//
+//@CLASSES:
+//  bmqimp::SessionId: Cache the instance number
+//
+//@DESCRIPTION: Overload operator<< as a single point for logging the number.
+//
+/// Thread Safety
+///-------------
+// Thread safe.
+
+#include <bmqp_ctrlmsg_messages.h>
+
+// BDE
+#include <bslim_printer.h>
```

Comment:
The component declared in this header allows to print with indentation. Its features are not used in this class. Possible to replace with more specific include that declares `bsl::ostream`